microsetta_interface/implementation.py

```diff
@@ -1203,6 +1205,20 @@ def get_ajax_list_kit_samples(kit_name):
     return flask.jsonify(result), code
 
 
+def get_ajax_check_activation_code(code, email):
+    response = requests.get(
+        ApiRequest.API_URL + '/can_activate',
+        auth=BearerAuth(session[TOKEN_KEY_NAME]),
+        verify=ApiRequest.CAfile,
+        params=ApiRequest.build_params({"email": email, "code": code}))
```

Comment:
I don't think it matters if the email address is sent, just that it's a valid code?

Reply:
...if the email is sent, then we would need to prompt the user to make sure they use the same email for which they received an activation code for